### DISCUSSION TITLE
Toggle and Search icon not responding

### DISCUSSION THREAD

**Discussion by elishaadamu** [cotes2020/jekyll-theme-chirpy#1626] [Q&A] [answered]
### Checklist

- [X] I have read and accepted the [contributing guidelines](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/docs/CONTRIBUTING.md).

### What is the topic?

Sharing tips and tricks

### Description

The search icon and hamburger toggle icon does not work in mobile mode, any ideas to resolve this issue:

Here are the links to the repository and web link.
Repository: https://github.com/elishadamu97/my-chirpy-site
Web link: https://elishadamu97.github.io/my-chirpy-site/

**Comment by kungfux** [comment, marked as answer]
It does not work because the script responsible for that does not exist in your `dist` folder.

```http
GET https://elishadamu97.github.io/my-chirpy-site/assets/js/dist/home.min.js
404 Not Found
```

If you download [build artifact](https://github.com/elishadamu97/my-chirpy-site/actions/runs/8401770092) you will see there is no such file in `assets/js/dist` folder.

For your reference, I have many other files there:
```bash
\assets\js\dist\app.min.js
\assets\js\dist\categories.min.js
\assets\js\dist\commons.min.js
\assets\js\dist\home.min.js
\assets\js\dist\misc.min.js
\assets\js\dist\page.min.js
\assets\js\dist\post.min.js
```

**Reply by elishaadamu** [reply]
how do I locate this files. I can't locate it in the chirpy theme

**Reply by kungfux** [reply]
Normally, they are being generated in CI during build step.

Locally, you can generate them by executing `npm run build` command.

**Reply by elishaadamu** [reply]
Got this error: 
> jekyll-theme-chirpy@6.5.3 build
> NODE_ENV=production npx rollup -c --bundleConfigAsCjs

'NODE_ENV' is not recognized as an internal or external command,
operable program or batch file.

**Reply by kungfux** [reply]
Please refer to [development documentation](https://github.com/cotes2020/jekyll-theme-chirpy/wiki/Development-&-Test-Environments).

However, I would recommend you to consider using [Chirpy Starter](https://chirpy.cotes.page/posts/getting-started/#option-1-using-the-chirpy-starter) template instead.
